### ISSUE TITLE
Feature: not= should trigger 'Write expected value first' warning like =

### ISSUE THREAD

**Issue by jramosg** [clj-kondo/clj-kondo#2739]
- [x] I have read the [Clojure etiquette](https://clojure.org/community/etiquette) and will respect it when communicating on this platform.

## Is your feature request related to a problem? Please describe.

The `not=` operator does not trigger the "Write expected value first" warning, while `=` does.

**Example:**

```clojure
;; This triggers a warning (as expected):
(defn foo [x] (if (= x 1) :one :other))
;; <stdin>:1:24: warning: Write expected value first

;; This does NOT trigger a warning (but should):
(defn foo [x] (if (not= x 1) :other :one))
;; linting took 58ms, errors: 0, warnings: 0
```

## Describe the solution you'd like

`not=` should trigger the same "Write expected value first" warning as `=` when the variable appears before the literal/constant value. The expected code would be:

```clojure
(defn foo [x] (if (not= 1 x) :other :one))
```

## Describe alternatives you've considered

- Leaving it as-is, but this creates inconsistency in the linting rules

## Additional context

This appears to be a gap in the linter's rule implementation. The same principle that applies to `=` (constants/literals should come first for readability and to avoid accidental assignment-like patterns from other languages) should also apply to `not=`.

I will submit a PR
